.github/workflows/update-openapi.yml

```
@@ -0,0 +1,159 @@
+            echo "create_pr=true" >> $GITHUB_OUTPUT
+          fi
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Create branch and PR if needed
+        if: steps.fetch-spec.outputs.has_changes == 'true' && steps.check-prs.outputs.create_pr == 'true'
+        run: |
+          # Configure Git
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)